examples/images/diffusion/README.md

```diff
@@ -96,11 +96,55 @@ We provide the finetuning example on CIFAR10 dataset
 
 You can run by config `train_colossalai_cifar10.yaml`
 ```
-python main.py --logdir /tmp -t --postfix test -b configs/train_colossalai_cifar10.yaml
+python main.py --logdir /tmp -t --postfix test -b configs/train_colossalai_cifar10.yaml 
+```
+
+## Inference
+you can get yout training last.ckpt and train config.yaml in your `--logdir`, and run by
```

Comment:
You can generate training checkpoints including last.ckpt and train config.yaml to the directory passed by `--logdir`?

Reply:
yes, the ckpt will be in logdir/{time}-{train.yaml)/checkpoint/last.ckpt